samples_project/Assets/SampleViewer/Samples/VRSample/readme.md

```diff
@@ -25,7 +14,21 @@ This sample runs off of the OpenXR framework and supports the following headsets
 6. HTC Vive Cosmos
 7. Valve Index
 
-In order to use this sample, connect your headset to your computer and ensure it is detected properly. Upon loading the sample, place the headset on your head. It should automatically use the headset to control the cameras height and rotation as well as position in the world. To move, use the joystick on the right controller, pushing the joystick will move you in that direction. The joystick on the left controller can be used to 'snap turn' 45 degrees, making it so you can turn around quickly.
+1. Place the headset on your head. The sample should automatically use the headset to control the camera's position and rotation in the world. 
+2. To move, use the joystick on the left controller, pushing the joystick will move you in that direction. The joystick on the right controller can be used to 'snap turn' 45 degrees, making it so you can turn around quickly. Additionally, pushing the right joystick up or down will allow the player to fly up or down in the world. 
+
+Please ensure that the project is using URP if you are using any of the Oculus Headsets listed above. HDRP may be used for any of the other headsets, but not Oculus. It will cause errors in the project. 
+
+## How it works
+
+1. Check the box for **Mesh Colliers Enabled** on the `ArcGIS Map View` component if you want the camera to be able interact with buildings and the ground. This will have a negative impact on performance so you may want to disable this on less powerful devices.
+2. Drag and Drop the VR Origin Prefab into the scene.
+3. Drag and drop the XR Interaction Manager Prefab into the scene.
+4. Draga an drop the Locomotion System Prefab into the scene.
+5. Parent the XR Origin prefab to the ArcGISMap game object.
+6. Expand the XR Origin fully and select the Main Camera Game Object.
+7. In the Inspector, click add component and search "ArcGIS Camera Component" and add it to the Main Camera Game Object.
+8. Press play and enjoy exploring.
```

Comment:
This step looks more like How to use the sample and not How the sample is created to show the capability... If so, delete this line.